usr/share/rear/conf/default.conf

```diff
@@ -3916,6 +3916,9 @@ UEFI_BOOTLOADER=""
 # so when for example SECURE_BOOT_BOOTLOADER="/boot/efi/EFI/BOOT/shim.efi" is specified
 # then all /boot/efi/EFI/BOOT/grub*.efi files are made available as second stage bootloader.
 # For more details see the output/ISO/Linux-i386/250_populate_efibootimg.sh script.
+#
+# If the mokutil tool is available on the system, and if it reports that secure boot is enabled,
+# then ReaR will try to guess the shim.efi and set it as SECURE_BOOT_BOOTLOADER.
```

Comment:
I realized that there is a small behavior change: previously, if SECURE_BOOT_BOOTLOADER was null, it meant not to use the shim. Now, it means to detect whether Secure Boot is used. Probably this is fine, because if we detect Secure Boot, it does not make much sense to not use the shim, as this is almost guaranteed to break. Still, it does not allow one to override the autodetection.

In my PR I chose the approach that unset `SECURE_BOOT_BOOTLOADER` means to autodetect, null means to avoid using shim (but my autodetection is different and detect whether shim is in use, not whether Secure Boot is turned on).

https://github.com/rear/rear/pull/3277/files#diff-00613392f8fca2a4c8c6ffb1eb59e66f4e1504eeeb14cf6b5a13458310a8227eR3814